### ISSUE TITLE
Preferred Google Acccount

### ISSUE THREAD

**Issue by cptramzis** [sanderheilbron/seo-shortcuts#10]
Hi @sanderheilbron,

I've been a heavy user of "SEO Peek" for a while and just recently rediscovered "SEO shortcuts" again. It's absolutely amazing and mostly replaced my habit of using bookmarks to easily reach various SEO tools. Thanks!

I've a few ideas for further development:
- _gsc_ - could lead to a "URL Inspect" of the new version of GSC.
- there could be an input in the settings to predefine ID of preferred Google Account. Currently after typing "gsc" I'm being redirected to my default Google Account (which is my personal account). Most of my client accounts are accesible via my business account only.

Account ID is a parameter in the URL:
https://search.google.com/u/1/search-console/inspect?resource_id=https://www.example.com

**Comment by MartijnOud** [comment]
Good idea! I'm logged in to multipe Google Accounts by default as well and currently use bookmarks to manage the different accounts. This can also be used for other Google sites like Analytics and AdWords.

Just thinking, but maybe something like this?

* `ga` --> default account (or based on setting)
* `ga2` --> `https://analytics.google.com/analytics/web/?authuser=2`
* `gsc4` --> `https://www.google.com/webmasters/tools/home?hl=nl&authuser=4`

**Comment by cptramzis** [comment]
Actually, that sounds even better as it adds more flexibility in different use cases. Looking forward to it!
